### ISSUE TITLE
above 20 open files with selections in each file stalls the Cockpit GUI

### ISSUE THREAD

**Issue by elcybob** [BjarniPeturFridjonsson/Qlik-SupportTools#2]
when opening above 20logs and select a random selection in each of the 20 documents the GUI is stalling

**Comment by BjarniPeturFridjonsson** [comment]
Yes, it stalles when we have total logfiles data that is greater than ~2 gigabites in memory. 

It was always planned to change it from reading the logfiles into memory and use the disk IO instead.
